skops/card/_model_card.py

```diff
@@ -380,6 +380,13 @@ def _generate_card(self) -> ModelCard:
         # add evaluation results
 
         template_sections = copy.deepcopy(self._template_sections)
+        if self.metadata:
+            model_file = self.metadata.to_dict().get("model_file", None)
```

Comment:
nit: not required

```suggestion
            model_file = self.metadata.to_dict().get("model_file")
```

Reply:
They can already add the section if they want to with `.add()`, it looks a bit ugly IMO that's why I removed it.